crates/locutus-node/src/node/event_listener.rs

```diff
@@ -234,6 +262,17 @@ mod test_utils {
             })
         }
 
+        pub fn get_broadcast_count(
+            &self,
+            expected_key: &ContractKey,
+            expected_value: &ContractValue,
+        ) -> usize {
+            let mut logs = self.logs.read();
+            logs.iter().filter(|log| {
```

Comment:
if we are doing this for testing pourpouses, wouldn't it be more correct to check that the broadcast has been received by the target nodes instead?

Reply:
Yes, I had thought to do that, the problem is that right now the target node of the broadcast is random. Another option that I have thought of is to verify that the target node is one of the two nodes to which should be broadcast.

Reply:
Yeah is enough to verify that any node (whichever it is) has received the broadcast (for now at least, eventually we would have to verify that multiple broadcasts have happened for different nodes when initiating one put). If you can change that we can merge this.